packages/code-migrate/src/Migration.ts

```diff
@@ -91,24 +96,16 @@ You must supply a createFunction as the third argument`
     after: this.after,
   };
 
-  getMigrationInstructions(): FileAction[] {
-    return this.instructions;
-  }
-
   write() {
     this.fs.writeChangesToDisc();
-  }
-
-  run() {
-    this.write();
-
-    // run after write hooks
     this.afterHooks.forEach((fn) => fn());
   }
 
-  static create(options: Options): Migration {
+  static init(options: Options): Migration {
```

Comment:
We have too many "create" methods in this project, it makes it easier to find this when needed to jump to symbol.